src/engraving/dom/fret.cpp

```diff
@@ -250,7 +248,7 @@ std::vector<LineF> FretDiagram::dragAnchorLines() const
 //---------------------------------------------------------
 
 void FretDiagram::setStrings(int n)
-{
+{   
```

Comment:
Please avoid adding whitespace at the end of a line, because that makes the code style check unhappy...